opm/simulators/linalg/bda/BILU0.hpp

```diff
@@ -115,11 +115,16 @@ class BILU0 : public Preconditioner<block_size>
         return rmat.get();
     }
 
-    void get_preconditioner_structure(std::vector<int>& pointers, std::vector<int>& indices, std::vector<int>& diagIndex_)
+    // void get_preconditioner_structure(std::vector<int>& pointers, std::vector<int>& indices, std::vector<int>& diagIndex_)
+    // {
+    //     pointers.assign(LUmat->rowPointers, LUmat->rowPointers + (Nb + 1));
+    //     indices.assign(LUmat->colIndices, LUmat->colIndices + nnzb);
+    //     diagIndex_ = diagIndex;
+    // }
```

Comment:
I guess we should remove the commented out code?

Reply:
Oops. Removed it already. I'll push soon.